docs/src/extends.md

```diff
@@ -154,13 +154,13 @@ It is also recommended that one also implements the following statistics functio
 - [`entropy(d::MultivariateDistribution)`](@ref)
 - [`cov(d::MultivariateDistribution)`](@ref)
 
-## Create a Matrix-variate Distribution
+## Create a matrix-variate Distribution
```

Comment:
It seems the whole page uses title case, so I guess we should rather change it to
```suggestion
## Create a Matrix-Variate Distribution
```
(at least that's the recommendation for title case of words with hyphen in e.g. JSS)